config/settings.yml

```diff
@@ -0,0 +1,2 @@
+# Shared settings across all environments: https://www.rubydoc.info/gems/config/5.1.0:
+# Overriden by: ./config/settings/#{environment}.yml
```

Comment:
Settings based on precedence with ./config/settings/#{environment}.yml (where @{environment} is the active Rails environment) overridding config/settings.yml.